File: view/src/main/java/by/rekuts/travelagency/view/controller/ControllerHelper.java
package by.rekuts.travelagency.view.controller;

import org.springframework.web.context.request.RequestContextHolder;
import org.springframework.web.context.request.ServletRequestAttributes;

import javax.servlet.http.HttpSession;

class ControllerHelper {
    int getSessionUserId() {
        var attr = (ServletRequestAttributes) RequestContextHolder.currentRequestAttributes();
        HttpSession session = attr.getRequest().getSession();
        return (int) session.getAttribute("sesUserId");
    }
}

File: view/src/main/scala/by/rekuts/travelagency/view/controller/ControllerHelper.scala
package by.rekuts.travelagency.view.controller

import org.springframework.web.context.request.{RequestContextHolder, ServletRequestAttributes}

class ControllerHelper {
  private[controller] def getSessionUserId = {
    val attr = RequestContextHolder.currentRequestAttributes.asInstanceOf[ServletRequestAttributes]
    val session = attr.getRequest.getSession
    session.getAttribute("sesUserId").asInstanceOf[Int]
  }
}
